### ISSUE TITLE
Explanation for the result。

### ISSUE THREAD

**Issue by unintendedly** [grip-unina/poi-forensics#8]
After using the test code, I have some questions about the results: What exactly do the scores in the outputs of POI-Forensics and ID-Reveal refer to? Are the scores next to the boxes in the output video the scores of the POIs? The calculation method of the POI score is not explained in detail in the paper, it only roughly mentions similarity. However, in my results, scores greater than 1 appeared, which contradicts the understanding of similarity. Hoping to get a response.Thanks.

**Comment by Wertiz** [comment]
The number you see next to the boxes in the output videos is the feature-wise sum of the euclidean distance between the audio under test and the closest element of the reference set.

**Comment by unintendedly** [comment]
Thank you for your reply. After obtaining this Euclidean distance, how do I determine if a video is forged?

**Comment by Wertiz** [comment]
Please refer to my comment on grip-unina/PoIForensics-Audio#1 as it answer a similar question. Feel free to ask more here if you have any more doubts.

**Comment by unintendedly** [comment]
I saw you mentioned that a csv file will be created, do you mean the npz file? Besides, is Global score the ultimate criterion used to determine the authenticity of videos as the 'score' you mentioned in https://github.com/grip-unina/PoIForensics-Audio/issues/1?

**Comment by Wertiz** [comment]
To avoid confusion, let me address each point:

1. To check whether a single video is real or fake you should compare its results with the results of several videos you know are real. Usually real videos have lower distances than fake ones. To observe this distance you should enable the `--create_plot 1` flag. This will produce a plot in which you can observe the distance value over time. If you observe that the fake video has distance values similar to all real ones, it means that either the reference set is not adequate and you should increase the variety/amount of real videos or that the fake video is well crafted and is not detectable using this technique.
2. This technique does not produce a csv file with all the distances. The npz file in the output directory contains several information, including the distance values. I would suggest using the plot to see the results as it is more user-friendly. 
3. Global score should be the mean of all the distances for that video. It gives you an idea but refer to all the values for a better understanding.
4. There is no absolute criterion to determine if a video is fake or real. This technique is a one class approach, meaning it knows the distribution of only the real class. You _could_ put a threshold to turn the distance score into a binary label but finding the absolute value for a threshold becomes a calibration issue and we did not address that in our work.

**Comment by unintendedly** [comment]
How to calculate AUC and Pd values based on the results of the program?

**Comment by unintendedly** [comment]
How to calculate AUC and Pd values based on the results of the program?

**Comment by Wertiz** [comment]
Please refer to [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score) for the function to calculate the AUC score.
The PD is the probability of detection (true positive rate) given one false positive value. You can easily calculate it by using [np.interp](https://numpy.org/doc/stable/reference/generated/numpy.interp.html) by giving the threshold, the false positive rate and the true positive rate.
